skins/wmde19/src/error.ts

```diff
@@ -0,0 +1,22 @@
+import Vue from 'vue';
+import Error from './Error.vue';
+
+Vue.config.productionTip = false;
+
+const dataElement = document.getElementById( 'app' );
+let applicationVars: any = {}; // TODO Create interface
+let messages = {};
+
+if ( dataElement ) {
+	applicationVars = JSON.parse( dataElement.dataset.applicationVars || '{}' );
+	messages = JSON.parse( dataElement.dataset.applicationMessages || '{}' );
+}
+
+new Vue( {
+	render: h => h( Error, {
+		props: {
+			errorMessage: applicationVars.message,
+			messages: messages,
```

Comment:
Isn't this property holding the error page messages? If yes, what's `errorMessage`? 🤔

Reply:
`messages` holds the translated i18n strings from `messages.json` in the content repository.
To avoid the confusion between `message` (i.e. error message) and `messages` (i.e. i18n string) I've called the property `errorMessage` in the component.

When we have an i18n mechanism, we can rename `errorMessage` to `message` again for greater consistency and less verbosity.